core/trino-main/src/main/java/io/trino/sql/planner/plan/TableWriterNode.java

```diff
@@ -208,28 +208,25 @@ public PlanNode replaceChildren(List<PlanNode> newChildren)
     {
         @Override
         public abstract String toString();
+
+        public abstract String getCatalogName();
```

Comment:
Did you consider returning `CatalogName` object here?

Reply:
Yeah, I did. But then I would need move the field catalogName to the `WriterTarget` and make it JSON-serializable. I did not change that :)

Reply:
Isn't the `CatalogName` class already JSON serializable?

Reply:
Oh, I misunderstood what do you mean earlier. Yeah, I can try to use `CatalogName`

Reply:
Applied.